google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -546,23 +514,19 @@ public void actionPerformed(ActionEvent event) {
 
     @Override
     protected void doLogin() {
-      Services.getLoginService().logIn(null, new IGoogleLoginCompletedCallback() {
-
-        @Override
-        public void onLoginCompleted() {
-          synchronize(true);
-        }
-      });
+      Services.getLoginService().logIn(null, () -> synchronize(true));
```

Comment:
parameter comments would be helpful here. (for instance that the closure is 'onLoginCompleted')